### DISCUSSION TITLE
2023/08/14/20.new_computer/

### DISCUSSION THREAD

**Discussion by giscus** [qllokirin/hexo_blog_comments#8] [Announcements]
# 2023/08/14/20.new_computer/

https://qllokirin.github.io/2023/08/14/20.new_computer/

**Comment by zzsqwq** [comment]
哈哈哈哈响应战sir号召泪目了，涵巨太强惹~~

**Comment by rabbitLove520** [comment]
我的配置
CPU: i5 13400
主板: TUF GAMING B660M-PLUS WIFI D4 
散热器: 利民 AK120 SE
电源	: 长城铜牌模组-750W
机箱：启航者A1 Pro
